### PR TITLE
(FACT-2503) Fix RSpec/NamedSubject offences

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: spec/custom_facts/core/execution/fact_manager_spec.rb [hunk 1]
```diff
@@ -134,44 +136,48 @@ def handy_method
     end
 
     describe 'and the command is not present' do
+      subject(:executor) { Facter::Core::Execution::Posix.new }
```
Comment (sebastian-miclea) [inline spec/custom_facts/core/execution/fact_manager_spec.rb]: Why is this needed? It's already defined.

Comment (oanatmaria) [reply spec/custom_facts/core/execution/fact_manager_spec.rb]: done

**Review by sebastian-miclea — APPROVED** [review]
